internal/service/ipam/model_ipamsvc_fixed_address.go

```diff
@@ -0,0 +1,240 @@
+	MatchType                 types.String      `tfsdk:"match_type"`
+	MatchValue                types.String      `tfsdk:"match_value"`
+	Name                      types.String      `tfsdk:"name"`
+	Parent                    types.String      `tfsdk:"parent"`
+	Tags                      types.Map         `tfsdk:"tags"`
+	UpdatedAt                 timetypes.RFC3339 `tfsdk:"updated_at"`
+}
+
+var IpamsvcFixedAddressAttrTypes = map[string]attr.Type{
+	"address":                      types.StringType,
+	"comment":                      types.StringType,
+	"created_at":                   timetypes.RFC3339Type{},
+	"dhcp_options":                 types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcOptionItemAttrTypes}},
+	"disable_dhcp":                 types.BoolType,
+	"header_option_filename":       types.StringType,
+	"header_option_server_address": types.StringType,
+	"header_option_server_name":    types.StringType,
+	"hostname":                     types.StringType,
+	"id":                           types.StringType,
+	"inheritance_assigned_hosts":   types.ListType{ElemType: types.ObjectType{AttrTypes: InheritanceAssignedHostAttrTypes}},
+	"inheritance_parent":           types.StringType,
+	"inheritance_sources":          types.ObjectType{AttrTypes: IpamsvcFixedAddressInheritanceAttrTypes},
+	"ip_space":                     types.StringType,
+	"match_type":                   types.StringType,
+	"match_value":                  types.StringType,
+	"name":                         types.StringType,
+	"parent":                       types.StringType,
+	"tags":                         types.MapType{ElemType: types.StringType},
+	"updated_at":                   timetypes.RFC3339Type{},
+}
+
+var IpamsvcFixedAddressResourceSchemaAttributes = map[string]schema.Attribute{
+	"address": schema.StringAttribute{
+		Required:            true,
+		MarkdownDescription: "The reserved address.",
+	},
+	"comment": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The description for the fixed address. May contain 0 to 1024 characters. Can include UTF-8.",
+	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Time when the object has been created.",
+	},
+	"dhcp_options": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: IpamsvcOptionItemResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: "The list of DHCP options. May be either a specific option or a group of options.",
+	},
+	"disable_dhcp": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "Optional. _true_ to disable object. The fixed address is converted to an exclusion when generating configuration.  Defaults to _false_.",
+	},
+	"header_option_filename": schema.StringAttribute{
```

Comment:
please add an UT for `header_option_filename`, `header_option_server_address`, `header_option_server_name`

Reply:
updated